packages/notification-center/src/hooks/useFeedUnseenCountQueryKey.ts

```diff
@@ -0,0 +1,12 @@
+import { useMemo } from 'react';
+import type { IStoreQuery } from '@novu/client';
+
+import { FEED_UNSEEN_COUNT_QUERY_KEY } from './queryKeys';
+import { useSetQueryKey } from './useSetQueryKey';
+
+export const useFeedUnseenCountQueryKey = (query?: IStoreQuery) => {
```

Comment:
Not sure I am following up, in this case, we can not get the query inside this hook from the useStore hook?

Reply:
@LetItRock

Reply:
this hook is used in the `useFeedUnseenCount` which is used for every feed tab, which passes it's `storeQuery` to the hook `useFeedUnseenCount` --> `useFeedUnseenCountQueryKey`